scalableminds/webknossos
# Decaf

Mailable description of changes (needs to be understandable bei webknossos mailing list people):
- All the coffeescript code was converted to javascript. Shouldn't change anything visible for the user.

Steps to test:
- npm test
- deploy and check interface

Issues:
- fixes #1562

------


This is the WIP state of the decaffination. One e2e test is still failing, I'm working on that.


<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1566/create?referer=github" target="_blank">Log Time</a>
